SYCL/Basic/get_backend.cpp

```diff
@@ -1,4 +1,4 @@
-// UNSUPPORTED: TEMPORARY_DISABLED
+// REQUIRES: TEMPORARY_DISABLED
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: env SYCL_DEVICE_FILTER=%sycl_be %t.out
```

Comment:
According to test logic it should be
```suggestion
// RUN: %t.out
```